components/Fields/Field.vue

```diff
@@ -21,6 +21,7 @@
       :class="`field_header_level_${recursionStack.length}`"
       >{{ fieldTranslateK(field.field) }}</FieldsHeader
     >
+    <Stars v-if="stars" :stars="stars"></Stars>
   </AddressField>
```

Comment:
Not into AddressField!

```suggestion
  </AddressField>

  <Stars v-if="stars" :stars="stars"></Stars>
```

Reply:
i tried to add it outside of the other fields, but Vue complains about having only one root element.
i can wrap all fields in a div to have only one root element, other fields seems to be mutually exclusive

Reply:
This one is also mutually exclusive : `v-else-if="field.field == 'starts'"`